scripts/render_configs.py

```diff
@@ -36,8 +40,16 @@
     supervisor_rendered = supervisor_conf.render(
       {'deployment_id': deployment_id})
 
+  with open(crontask_template_path) as f:
```

Comment:
I have never noticed before that this file has only 2 spaces! We should fix that at some point... I'll leave it to you whether you want to do it in this PR.

Reply:
Fixed! The spacing is fine in the [template](https://github.com/datamade/deploy-a-site/blob/master/templates/django/%7B%7Bcookiecutter.directory_name%7D%7D/scripts/render_configs.py), so I'm not sure how that happened.